### PR TITLE
Tweak colors

### PR DESCRIPTION
**Pull request by CyberShadow**
Implement MattCoder's mockup from
http://forum.dlang.org/post/dfflvgdzanirmsqgxgxz@forum.dlang.org

### CODE REVIEW

File: css/cssmenu.css.dd [hunk 1]
```diff
@@ -177,10 +178,11 @@ _=
...
 menu_a=#ffffff
 menu_a_border=$(martian_red_light)
-menu_a_shadow=#751d0c
-menu_bg=$(martian_red)
+menu_a_shadow=#444
+menu_active_bg=#c33
+menu_bg=#666
 menu_bg2=$(menu_bg)
-menu_bg_hover=$(martian_red_light)
+menu_bg_hover=#777
```
Comment (andralex) [inline css/cssmenu.css.dd]: could you please put this in a variable

Comment (CyberShadow) [inline css/cssmenu.css.dd]: This is the macro section already, what do you mean?

**Comment by andralex** [comment]
rebase... also please ask for opinions in the forum

**Comment by CyberShadow** [comment]
Rebased.

> also please ask for opinions in the forum

Two people said they like this change:

  - http://forum.dlang.org/post/zclhxljrjktbaqxeypiy@forum.dlang.org
  - http://forum.dlang.org/post/mpydaqqreadidqpjrcij@forum.dlang.org

**Comment by CyberShadow** [comment]
Actually this is not ready to merge, it clashes with some recent pulls.

**Comment by CyberShadow** [comment]
OK, updated with some nice tweaks. I think you'll like this :)

**Comment by andralex** [comment]
I'll just trust you

**Pull request merged**